lib/adBlockRustUtils.js

```diff
@@ -11,6 +11,32 @@ const uBlockScriptlets = path.join(uBlockLocalRoot, 'assets/resources/scriptlets
 
 const braveResourcesUrl = 'https://raw.githubusercontent.com/brave/adblock-resources/master/dist/resources.json'
 
+const defaultListsUrl = 'https://raw.githubusercontent.com/brave/adblock-resources/master/filter_lists/default.json'
+const regionalListsUrl = 'https://raw.githubusercontent.com/brave/adblock-resources/master/filter_lists/regional.json'
+
+/**
+ * Returns a promise that which resolves with the body parsed as JSON
+ *
+ * @param url The URL to fetch from
+ * @return a promise that resolves with the content of the list or rejects with an error message.
+ */
+const requestJSON = (url) => new Promise((resolve, reject) => {
```

Comment:
This would be nice to write as a standard async function.

Also also, pet peeve but having two functions declared on the same line is all sorts of confusing :)

Reply:
This is just copied 100% from where it used to be in `scripts/generateAdBlockRustDataFiles.js` - although I believe it's done this way because it needs to wrap the `request` API and also take a `url` argument.